src/main/cpp/fielddata/DragonTargetFinder.cpp

```diff
@@ -150,6 +151,37 @@ optional<tuple<DragonTargetFinderData, Pose2d>> DragonTargetFinder::GetPose(Drag
             return make_tuple(DragonTargetFinderData::ODOMETRY_BASED, fieldconst->GetFieldElementPose(FieldConstants::FIELD_ELEMENT::BLUE_REEF_CENTER).ToPose2d());
         }
     }
+    else if (item == DragonTargetFinderTarget::BARGE)
+    {
+        auto allianceColor = FMSData::GetInstance()->GetAllianceColor();
+        auto bargeHelper = BargeHelper::GetInstance();
+        if (allianceColor == frc::DriverStation::Alliance::kRed)
```

Comment:
Why isn't barge helper taking care of alliance color?  I don't think target finder should have this logic